Packs/GigamonThreatINSIGHT/Integrations/GigamonThreatINSIGHT/GigamonThreatINSIGHT.py

```diff
@@ -400,6 +401,80 @@ def formatEvents(r_json):
     return newData
 
 
+def getFirstFetch(firstFetch) -> dict[str, Any]:
+
+    pieces = firstFetch.strip().split()
+    unit = pieces[len(pieces) - 1]
+
+    try:
+        delta = arg_to_number(
+            arg=pieces[0],
+            arg_name='first_fetch',
+            required=False
+        )
+    except Exception:
+        delta = 0
+
+    days: int = delta if delta and unit == 'days' else 0
+    hours: int = delta if delta and unit == 'hours' else 0
+
+    if days == 0 and hours == 0:
+        days = 7
+
+    return {'days': days, 'hours': hours}
+
+
+def mapSeverity(severity) -> int:
+    match severity:
+        case 'high':
+            return 3
+        case 'moderate':
+            return 2
+        case 'low':
+            return 1
+        case _:
+            return 0
```

Comment:
Do we know for sure 0 would be valid for Cortex and not cause a crash/exception on Cortex side. Would returning 1 be safer?

Reply:
Cortex state in (https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-6/cortex-xsoar-admin/incidents/incident-management) that Incident severity levels are: Unknown (0), Informational (0.5), Low (1), Medium (2), High (3), Critical (4). 
That is why I used 0 as the default value